### ISSUE TITLE
Pudge Weapon Not Synced

### ISSUE THREAD

**Issue by ghost** [pissang/dota2hero#3]
The hook on Pudge isn't synced with his body, it can be seen slightly to the right above his head.

![hook](https://f.cloud.github.com/assets/4075747/2459598/6b0089f4-af5e-11e3-9be1-51e1590a548c.png)

**Comment by pissang** [comment]
Yes, and necrolyte has the same problem. It seems some animation data is broken after extracted from mdl file
